_studio/mfx_lib/mctf_package/mctf/src/mctf_common.cpp

```diff
@@ -2329,7 +2330,7 @@ mfxU16 CalcNoiseStrength(
 
     s  = c3 * pow(STC, 3.0) + c2 * pow(STC, 2.0) + c1 * STC + c0;
     s2 = d3 * pow(ISTC, 3.0) + d2 * pow(ISTC, 2.0) + d1 * ISTC + d0;
-    s = NMIN(s, s2) + 4;
+    s = NMIN(s, s2) + 5;
```

Comment:
Is this change by intent?

Reply:
Yes, this is for compression efficiency improvement.